datasets/br/br_tcu_disqualified/br_tcu_disqualified.yaml

```diff
@@ -0,0 +1,24 @@
+title: Brazil's TCU list of individuals disqualified from public service
```

Comment:
Remember the directory structure convention is br/tcu_disqualified/br_tcu_disqualified.yaml i.e. no country code in second level